README.md

```diff
@@ -1,83 +1,65 @@
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
 
-To run the website locally when you have Hugo installed:
+## Contributing to the docs
 
-```bash
-make serve
-```
+You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account. This copy is called a *fork*. Make any changes you want in your fork, and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
 
-This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear, actionable feedback.  As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
 
-## Community, discussion, contribution, and support
+Also, note that you may end up having more than one Kubernetes reviewer provide you feedback or you may end up getting feedback from a Kubernetes reviewer that is different than the one initially assigned to provide you feedback.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+Furthermore, in some cases, one of your reviewers might ask for a technical review from a Kubernetes tech reviewer when needed.  Reviewers will do their best to provide feedback in a timely fashion but response time can vary based on circumstances.
 
-You can reach the maintainers of this project at:
+For more information about contributing to the Kubernetes documentation, see:
 
-- [Slack](https://kubernetes.slack.com/messages/sig-docs)
-- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+* [Contribute to Kubernetes docs](https://kubernetes.io/docs/contribute/)
+* [Using Page Templates](https://kubernetes.io/docs/contribute/style/page-templates/)
+* [Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
+
+## Localization `README.md`'s
+
+| Language  | Language |
+|---|---|
+|[French](README-fr.md)|[Korean](README-ko.md)|
+|[German](README-de.md)|[Portuguese](README-pt.md)|
+|[Hindi](README-hi.md)|[Spanish](README-es.md)|
+|[Indonesian](README-id.md)|[Chinese](README-zh.md)|
+|[Japanese](README-ja.md)|[Vietnamese](README-vi.md)|
+|[Russian](README-ru.md)|[Italian](README-it.md)|
+|[Polish](README-pl.md)|[Ukrainian](README-uk.md)|
```

Comment:
I think the idea was to have alphabetical order, columns first. If so, Korean should be after Japanese in column 1.